src/apis/user/dto/create-user.input.ts

```diff
@@ -2,30 +2,24 @@ import { Field, InputType } from '@nestjs/graphql';
 
 @InputType()
 export class CreateUserInput {
-  @Field()
-  name: string;
-
-  @Field()
-  nickname: string;
-
-  @Field()
+  @Field(() => String, {
+    description: '이메일',
+  })
   email: string;
 
-  @Field()
+  @Field(() => String, {
+    description: '비밀번호',
+  })
   password: string;
 
-  @Field()
-  cPassword: string;
-
-  @Field()
-  gender: string;
-
-  @Field()
-  age: string;
+  @Field(() => String, {
+    description: '닉네임',
+  })
+  nickname: string;
 
-  @Field(() => String)
-  phone_number: string;
+  @Field(() => String, { nullable: true, description: '휴대폰번호' })
+  phoneNumber: string;
```

Comment:
```suggestion
  phoneNumber: string | null;
```
nullable 한 필드라면 nullable 타이핑도 습관들이면 좋습니다.

Reply:
`string | null `을 해주면 `nullable: true ` 을 제거해도 되는거죠??